cli/required.go

```diff
@@ -43,6 +43,23 @@ func RequiresMinArgs(min int) cobra.PositionalArgs {
 	}
 }
 
+// RequiresMaxArgs returns an error if there is not at most min args
+func RequiresMaxArgs(min int) cobra.PositionalArgs {
```

Comment:
oh damn you're absolutely right :sweat: 
/me was in *robot* mode I guess :sweat_smile: